protocols/mdns/CHANGELOG.md

```diff
@@ -10,9 +10,12 @@
 
 - Removed the `lazy_static` dependency. See [PR 2977].
 
+- Update to `if-watch` `v2.0.0`. See [PR 2978].
```

Comment:
```suggestion
- Update to `if-watch` `v2.0.0` and thus the `async` method `Mdns::new` and `TokioMdns::new` becomes synchronous. See [PR 2978].
```

Might be helpful for people upgrading, though I guess the compile time error will help more.